packages/kbn-alerts-ui-shared/src/maintenance_window_callout/index.tsx

```diff
@@ -66,6 +89,32 @@ export function MaintenanceWindowCallout({
     });
   }, [categories, activeMaintenanceWindows]);
 
+  const categoryNames = useMemo(() => {
+    const activeCategoryIds = Array.from(
+      new Set(
+        activeMaintenanceWindows
+          .map(({ categoryIds }) =>
+            // If the categories array is provided, only display category names that are included in it
+            categoryIds?.filter((categoryId) => !categories || categories.includes(categoryId))
+          )
+          .flat()
+      )
+    );
+    const activeCategories = activeCategoryIds
+      .map(
+        (categoryId) =>
+          Object.values(APP_CATEGORIES).find((c) => c.id === categoryId)?.label ?? categoryId
+      )
+      .filter(Boolean) as string[];
+    return activeCategories.length === 0
+      ? null
+      : activeCategories.length === 1
+      ? `${activeCategories}`
```

Comment:
Should it be like that?
```suggestion
      ? `${activeCategories[0]}`
```

Reply:
An array with a length of `1` converts to a string equivalent to `array[0]`